### PR TITLE
add code

### PR DESCRIPTION
**Pull request by 5cotts**

### CODE REVIEW

**Review by lodrion — COMMENTED** [review]
can you rename directory "python" because potentially we'll end up adding other languages...

**Comment by 5cotts** [comment]
In that case, should each language/directory have it's own language specific README?

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+`wirescreen-client` is a custom client written to interact with the API which supports
  https://wirescreen.ai.
+ 
+ The below document shows how to use Python version of this client.
 
 ## Initializing the client
 The client token can be declared as a variable and passed directly to the client.
 ```python
-from wirescreen_api.client import WireScreenAPI
+from python.client import WireScreenAPI
```
Comment (lodrion) [inline README.md:11]: don't do this, people would be installing the package itself, not the python part...

Comment (5cotts) [reply README.md:11]: So should this be `from wirescreen_client.client import WireScreenAPI` or should I just remove the imports entirely from this README?

Comment (lodrion) [reply README.md:11]: just remove it

**Review by lodrion — APPROVED** [review]

**Pull request merged**